docs/.storybook/main.js

```diff
@@ -1,6 +1,18 @@
 /** @type {import("@storybook/core/types/index").StorybookConfig} */
 module.exports = {
 	stories: ["../src/**/*.stories.@(ts|tsx|mdx)"],
-	addons: ["@storybook/addon-docs", "storybook-css-modules-preset"],
```

Comment:
Also it's a good thing we can drop the "storybook-css-modules-preset". It's unofficial and not really stable. CSS Modules can now be set in the postcss plugin